src/MiNET/MiNET/Worlds/Level.cs

```diff
@@ -898,6 +898,16 @@ public void BreakBlock(Player player, BlockCoordinates blockCoordinates)
 			if (OnBlockBreak(new BlockBreakEventArgs(player, this, block, drops)))
 			{
 				block.BreakBlock(this);
+				
+				var lvlEvent = new McpeLevelEvent
+                {
```

Comment:
@Samueljh1 you're literally the only person on the whole planet that cares about identing and grammar